poi-scratchpad/src/main/java/org/apache/poi/hssf/converter/ExcelToHtmlConverter.java

```diff
@@ -692,6 +692,24 @@ public void processWorkbook(HSSFWorkbook workbook) {
         htmlDocumentFacade.updateStylesheet();
     }
 
+    public void processWorkbookSheet(HSSFWorkbook workbook, int sheetNumber) {
```

Comment:
Can you call this processWorksheet?

Reply:
would it be better to just pass a HSSFWorksheet in the input instead of a workbook and sheetNumber?